src/edap/constants.py

```diff
@@ -1,20 +1,253 @@
 
-
+# ISO3166-1-Alpha-2 as a key, CLDR display name as value
 COUNTRIES_CODES = {
+    "tw": "Taiwan",
+    "af": "Afghanistan",
+    "al": "Albania",
+    "dz": "Algeria",
+    "as": "American Samoa",
+    "ad": "Andorra",
+    "ao": "Angola",
+    "ai": "Anguilla",
+    "aq": "Antarctica",
+    "ag": "Antigua & Barbuda",
+    "ar": "Argentina",
+    "am": "Armenia",
+    "aw": "Aruba",
+    "au": "Australia",
     "at": "Austria",
+    "az": "Azerbaijan",
+    "bs": "Bahamas",
+    "bh": "Bahrain",
+    "bd": "Bangladesh",
+    "bb": "Barbados",
+    "by": "Belarus",
+    "be": "Belgium",
+    "bz": "Belize",
+    "bj": "Benin",
+    "bm": "Bermuda",
+    "bt": "Bhutan",
+    "bo": "Bolivia",
+    "bq": "Caribbean Netherlands",
+    "ba": "Bosnia",
+    "bw": "Botswana",
+    "bv": "Bouvet Island",
+    "br": "Brazil",
+    "io": "British Indian Ocean Territory",
+    "vg": "British Virgin Islands",
+    "bn": "Brunei",
+    "bg": "Bulgaria",
+    "bf": "Burkina Faso",
+    "bi": "Burundi",
+    "cv": "Cape Verde",
+    "kh": "Cambodia",
+    "cm": "Cameroon",
+    "ca": "Canada",
+    "ky": "Cayman Islands",
+    "cf": "Central African Republic",
+    "td": "Chad",
     "cl": "Chile",
-    "cz": "Czech Republic",
-    "de": "Germany",
+    "cn": "China",
+    "hk": "Hong Kong",
+    "mo": "Macau",
+    "cx": "Christmas Island",
+    "cc": "Cocos (Keeling) Islands",
+    "co": "Colombia",
+    "km": "Comoros",
+    "cg": "Congo - Brazzaville",
+    "ck": "Cook Islands",
+    "cr": "Costa Rica",
+    "hr": "Croatia",
+    "cu": "Cuba",
+    "cw": "Curaçao",
+    "cy": "Cyprus",
+    "cz": "Czechia",
```

Comment:
Nice one, we are indeed officially Czechia for some time now :-)

Reply:
Even I was surprised when saw at first, although I haven't ever lived in Czechia :)